pages/youth-board.vue

```diff
@@ -1,19 +1,21 @@
 <template>
   <div>
     <div class="container">
-      <h1 class="page-title">Vatterott Foundation Youth Board</h1>
-      <p
-        class="page-description"
-      >In 2001 the Vatterott Foundation extended its tradition of giving by forming the Vatterott Foundation Youth Board (VFYB). The Youth Board is comprised of Vatterott descendants aged 13 to 30 who are interested in giving back to their community in the spirit of their ancestors.</p>
+      <h1 class="page-title">{{ titlesection.title }}</h1>
+      <p class="page-description">
+        {{ titlesection.description }}
+      </p>
       <blockquote class="big-quote">
-        <p>Youth leading and supporting change to improve human conditions, all for the greater glory of God!</p>
+        <p>
+          {{ titlesection.blockquote }}
+        </p>
         <footer>
-          <cite class="author">Mission Statement</cite>
-          <cite class="citation-title">Vatterott Foundation Youth Board</cite>
+          <cite class="author">{{ titlesection.author }}</cite>
+          <cite class="citation-title">{{ titlesection.citation }}</cite>
         </footer>
       </blockquote>
       <section id="vfyb-about">
-        <h2 class="page-section-header">About the VFYB</h2>
+        <h2 class="page-section-header">{{ aboutVFYB.about }}</h2>
         <figure>
           <iframe
             src="https://player.vimeo.com/video/73003706"
```

Comment:
Let's add this video URL to markdown as well so authors can change it

Reply:
Done